Makefile

```diff
@@ -23,6 +23,19 @@ in: inplace # just a shortcut
 inplace:
 	$(PYTHON) setup.py build_ext -i
 
+meson:
```

Comment:
This does not really reflect that this install in developer / editable / inplace / intree mode.

I would prefer a more explicit `Makefile` target.

```suggestion
dev-meson:
```

Reply:
or `meson-dev` (to be consistent with `meson-clean`).